javascript/javascript1/week2/flight.js

```diff
@@ -0,0 +1,19 @@
+
+function getFullName(firstName, lastName, useFormalName, gender) {
+
+    if (useFormalName==true && gender=="male"){
+        return "Lord " + " " + firstName + " " + lastName;
+    }
+    else if (useFormalName==true && gender=="female"){
+        return `Lady  ${firstName} ${lastName}`;
+    }
+    else {
+        return firstName + " " + lastName;
+    }
+ }
+
+const fullname1 = getFullName("Benja", "Hughes",true,"male");
+const fullname2 = getFullName("Anna", "nilasen",true,"female");
+
+console.log(fullname1);
+console.log(fullname2);
```

Comment:
Overall really nice solution. ✨

**A few comments & observations:** 
- In some return statements you are using string concatenation and in others you are using template literals. It is great to see you experiment with both.🔥 In the future it would be good to focus on consistency, and stick with one. 
- There are two times space between lady/lord and the firstName. Check it out and see if you can see why that might be. 😊
- What happens if the string inputs are empty?